### DISCUSSION TITLE
python版本 ocr无法识别单个字符的图片

### DISCUSSION THREAD

**Discussion by wenheLuan** [PaddlePaddle/PaddleOCR#12561] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：windows11
- 版本号/Version：Paddle：2.6.0  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：
没有报错,识别结果是None
我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):
no
请尽量不要包含图片在问题中/Please try to not include the image in the issue.
![1708244177774](https://github.com/PaddlePaddle/PaddleOCR/assets/33254237/e71ec68f-8da4-4d41-8100-7c39e4d97cf9)

![1708244274729](https://github.com/PaddlePaddle/PaddleOCR/assets/33254237/2e09264f-9026-46b3-8aa6-be43040141a8)
![1708244336700](https://github.com/PaddlePaddle/PaddleOCR/assets/33254237/cc69e876-051a-431a-9699-488a38cfd340)

**Comment by SWHL** [comment]
这个可能原因是文本检测阶段就没检测到这个图像中文字。可以尝试以下两种方法：
1. 跳过检测，只要识别
2. 四周padding这张图，再过PaddleOCR
